vortex-array/src/array/varbinview/accessor.rs

```diff
@@ -16,20 +16,19 @@ impl ArrayAccessor<[u8]> for VarBinViewArray {
         let bytes: Vec<PrimitiveArray> = (0..self.metadata().buffer_lens.len())
             .map(|i| self.buffer(i).into_canonical()?.into_primitive())
             .try_collect()?;
+        let bytes_slices: Vec<&[u8]> = bytes.iter().map(|b| b.maybe_null_slice::<u8>()).collect();
```

Comment:
pre extracting slices from the arrays yeilds a modest 30% improvement when iterating over the array